src/utils/eventManager.ts

```
@@ -0,0 +1,122 @@
+import type { ErrorInfo } from './errorBoundary'
+
+/**
+ * Type-safe event map for medical calculator events
+ */
+export interface MedicalCalculatorEventMap {
+  'network:online': void
```

Biome (1.9.4)

[error] void is confusing outside a return type or a type parameter.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)